osu.Game/Screens/Menu/IntroTriangles.cs

```diff
@@ -330,9 +332,10 @@ private void load(LargeTextureStore textures)
             private partial class RulesetFlow : FillFlowContainer
             {
                 [BackgroundDependencyLoader]
-                private void load(RulesetStore rulesets)
+                private void load(IRulesetStore rulesets)
                 {
-                    AutoSizeAxes = Axes.Both;
+                    Width = 800;
```

Comment:
Should use max size from grid container for portrait / UI scale scenarios.

Will do after we get that second opinion.